sgkit/__init__.py

```diff
@@ -24,6 +26,12 @@
 from .testing import simulate_genotype_call_dataset
 from .window import window
 
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
```

Comment:
Might be nice to give ``__version__`` a default value here like ``"version not defined"`` or something, rather than having user code failing on an AttributeErrror?

Reply:
Fixed in https://github.com/pystatgen/sgkit/pull/376/commits/0e2e557014559b618e8c34c7c02bd929f2a8305f